core/src/components/datetime-button/datetime-button.scss

```diff
@@ -6,9 +6,13 @@
 :host {
   display: flex;
 
+  flex: 0 1 auto;
```

Comment:
Just noticed that this line doesn't seem to do anything

Reply:
You're right, [that is the initial value](https://developer.mozilla.org/en-US/docs/Web/CSS/flex#values)

I removed it here: https://github.com/ionic-team/ionic-framework/pull/27848/commits/924601f0d0b4f9b81da9a10604aa312cf665d6aa